ui/plugins/pluginpanel.html

```diff
@@ -174,9 +174,10 @@
             <span ng-switch-when='unknown'>{{job.machine_name}}</span>
             <a title="Open buildbot slave health report" target="_blank"
                href="{{ getSlaveHealthUrl(job.machine_name) }}"
-               ng-switch-default>
+               ng-switch-default ng-if="job.buildbot_request_id">
                {{job.machine_name}}
             </a>
+            <span ng-switch-default ng-if="!job.buildbot_request_id">{{job.machine_name}}</span>
```

Comment:
Agreed.  This approach would probably work fine, but better to be more explicit and use the field that really tells it like it is.  Also, this would be a good clue for the future If we ever wanted a "taskcluster-only" link.